modules/ROOT/pages/monitor-api-conformance.adoc

```diff
@@ -1,99 +1,164 @@
+You can view conformance details for governed APIs in their governance validation reports in API Governance and API Manager. See xref:find-and-fix-conformance-issues.adoc[].
 
 == Export Conformance Information
 
-You can export conformance information in CSV format from several places in the API Governance console. 
+You can export profile and API conformance information in CSV format from several places in the API Governance console. 
 
 * <<gov-profile-export>>
 * <<gov-api-export>>
 * <<gov-spec-profile-export>>
 
 [[gov-profile-export]]
-=== Governance Profile Export
-On the main API Governance console page, select the *Profiles* tab and then click the *Export Report* button to export governance profile information, such as: 
+=== API Conformance Information by Profile 
+
+To export profile information:
 
+. On the API Governance console, select the *Profiles* tab.
+. Click *Export Report*.
++
+This exports a CSV file with API conformance information for each profile in your organization. The information includes:   
++
 * Organization ID 
 * Profile IDs
 * Profile name
-* Status
+* Status (LIVE or DRAFT)
 * Total APIs validated
 * Number of conformant APIs (pass)
 * Number of nonconformant APIs (fail)
-* Timestamps
-* Associated ruleset tags
+* Timestamp
+* Associated tags in Exchange
 
 [[gov-api-export]]
-=== Governed API Export
-On the main API Governance console page, select the *Governed APIs* tab and then click *Export Report* to export the governed API information, such as:  
+=== Governed APIs Targeted by Active Profiles 
 
+To export a report of all governed APIs targeted by your active profiles:
+
+. On the API Governance console, select the *Governed APIs* tab.
+. Click *Export Report*. 
++
+This exports a CSV file with conformance information for each API targeted by the active profiles in your organization. The information includes:  
++
 * Organization ID 
-* API IDs
-* Versions
-* Total number of APIs
+* API ID
+* API Version
+* Total number of Asset Versions for that API Version
 * Number of conformant APIs (pass)
 * Number of nonconformant APIs (fail)
-* Timestamps
+* Timestamp
 
 [[gov-spec-profile-export]]
-=== Specific Profile Export
-On a selected governance profile page, click the more options menu icon (image:overflow-menu-icon.png[width=9,height=15,fit=line]) and then select *Export Reports* to export a report that shows the selected governance profile's information, such as: 
+=== API Conformance Information for a Specific Profile
+
+To export a conformance report for APIs targeted by a specific active or draft profile:
 
+. On the API Governance console, select the *Profiles* tab.
+. Select *Active* or *Draft*.
+. On a selected governance profile page, click the more options menu icon (image:overflow-menu-icon.png[width=9,height=15,fit=line]).
+. For the profile you want conformance information for, select *Export Report* from the more options menu. 
```

Comment:
Are 150 and 151 the same? Or does the more options menu list Export Report for multiple profiles?

Reply:
Good catch. I'll fix this.